### PR TITLE
Add Expenditure Trends and Estimates data story, adjusted for inflation by default

### PR DESCRIPTION
**Pull request by schalkventer**
image

image

### CODE REVIEW

**Comment by longhotsummer** [comment]
This is big and touches many files, github makes it hard to understand. Please confirm that 

* all charts still work as expected (display, hover, mobile, desktop, download)
* spot-check chart values for national and provincial departments for current and previous years

What happened to 404.html?

**Comment by schalkventer** [comment]
@longhotsummer The one in the diff is the previous default Jekyll one I removed. Our 404 is the `404.md` file.

File: 404.html [hunk 1]
```diff
@@ -1,24 +0,0 @@
----
```
[the remark this answers is not in the document]
Comment (schalkventer) [reply 404.html]: @longhotsummer The one in the diff is the default Jekyll one we had previously. Our 404 is the 404.md file.

**Comment by schalkventer** [comment]
@longhotsummer Can confirm the following:

> all charts still work as expected (display, hover, mobile, desktop, download)

**Comment by longhotsummer** [comment]
LGTM.

**Pull request merged**